src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -238,3 +240,47 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
 
         metrics.incr("github_pr_comment.api_error")
         raise e
+
+
+@instrumented_task(name="sentry.tasks.integrations.github_comment_reactions")
+def github_comment_reactions():
+    logger.info("github.pr_comment.reactions_task")
+
+    comments = PullRequestComment.objects.filter(
+        created_at__gte=datetime.now(tz=timezone.utc) - timedelta(days=30)
+    )
+
+    for comment in RangeQuerySetWrapper(comments):
+        pr = comment.pull_request
+        try:
+            repo = Repository.objects.get(id=pr.repository_id)
+        except Repository.DoesNotExist:
+            metrics.incr("github_pr_comment.comment_reactions.missing_repo")
+            continue
+
+        integration = integration_service.get_integration(integration_id=repo.integration_id)
+        if not integration:
+            logger.error(
+                "github.pr_comment.comment_reactions.integration_missing",
+                extra={"organization_id": pr.organization_id},
+            )
+            metrics.incr("github_pr_comment.comment_reactions.missing_integration")
+            return
+
+        installation = integration_service.get_installation(
+            integration=integration, organization_id=pr.organization_id
+        )
+
+        # GitHubAppsClient (GithubClientMixin)
+        # TODO(cathy): create helper function to fetch client for repo
+        client = installation.get_client()
+
+        try:
+            reactions = client.get_comment_reactions(repo=repo.name, comment_id=comment.external_id)
```

Comment:
You'll need to be mindful of [github's rate limits](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/rate-limits-for-github-apps#determining-rate-limits-for-a-github-app) here. We'll have a shared rate limit with other operations against GitHub could run out of quota, and need to retry.